### DISCUSSION TITLE
Request had invalid authentication credentials

### DISCUSSION THREAD

**Discussion by bajki** [freekmurze/laravel-analytics#421] [Q&A]
Hi, when I  set APP_CACHE=true i get this error.

{ "error": { "code": 401, "message": "Request had invalid authentication credentials. Expected OAuth 2 access token, login cookie or other valid authentication credential. See https://developers.google.com/identity/sign-in/web/devconsole-project.", "errors": [ { "message": "Invalid Credentials", "domain": "global", "reason": "authError", "location": "Authorization", "locationType": "header" } ], "status": "UNAUTHENTICATED" } }

With disabled cache is ok.

laravel: 8
laravel-analytics:4.0.1

In analytics.php 

 'cache_lifetime_in_minutes' => 0,

**Comment by danikaharu** [comment]
I also have the same problem. I have been using version 5. Can anyone provide a solution?
